README.md

```diff
@@ -25,3 +25,5 @@ export GITHUB_CREATE_REPO_ACCESS_TOKEN=xxxxx
 ```
 
 To be able to push to github without being prompted for username/password, set up git to use SSH
```

Comment:
Please remove the first `set up git to use SSH`, right before the link. Right now, it's rendered twice in the text :)